app-libs/stf/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2021"
 codec = { version = "3.0.0", default-features = false, features = ["derive"], package = "parity-scale-codec" }
 derive_more = { version = "0.99.5" }
 log = { version = "0.4", default-features = false }
+ring = { version = "0.16.20", default-features = false }
```

Comment:
Can you move it under `# litentry` and remove the `aes-gcm` dep?

Reply:
Done. 
`aes-gcm`, I tried to remove it before. Reported some strange dependency. Disappeared now.